### ISSUE TITLE
tower cli fails to list system auditor users

### ISSUE THREAD

**Issue by Akasurde** [ansible/tower-cli#689]
If I try - 

```
$ tower-cli user get --is-system-auditor True
Error: The Tower server claims it was sent a bad request.

GET https://xx.xx.xx.xx/api/v2/users/
Params: [('is_system_auditor', True)]
Data: None

Response: {"detail":"User has no field named 'is_system_auditor'"}
```

Can someone check if this is correct API usage ?

**Comment by sumkincpp** [comment]
Which AWX version it is?
I see that there are significant API changes from version to version.

**Comment by Akasurde** [comment]
sorry failed to mentioned this - Tower 3.4.3

**Comment by sumkincpp** [comment]
Unfortunately for me this doesn't work too on older AWX version(1.0.6.49). 
For me it is alsoe expected that field should be available for query since it's present in a model.

I believe this should work via API, but itself the field seems not be stored in Database
https://github.com/ansible/awx/commit/390ac656fab1ac0d5a26364de86ce98415d24ad7

**Comment by jainnikhil30** [comment]
I think this is issue awx/tower issue, `is_system_auditor` is not a filterable field. This would need a schema change on awx/tower. Not a tower-cli issue at this point, can we file this on awx ?  
```
"is_system_auditor": {
                "type": "boolean",
                "required": false,
                "label": "Is system auditor",
                "filterable": false,
                "default": false
```

**Comment by AlanCoding** [comment]
That's true, this is not a filterable field because it works via the role membership. In the API browser, you can try `/api/v2/users/?roles__role_field=system_auditor`, and that should do what you are trying to do with the non-filterable field. The field does not actually exist directly on the model.

You could also probably use some role related command in tower-cli.
